app/com/m3/octoparts/aggregator/handler/HttpPartRequestHandler.scala

```diff
@@ -87,7 +87,7 @@ trait HttpPartRequestHandler extends Handler {
       def method = httpMethod
       val uri = new URI(buildUri(hArgs))
       val maybeBody = hArgs.collectFirst {
-        case (p, values) if p.paramType == ParamType.Body && values.nonEmpty => values.head
+        case (p, headValue :: _) if p.paramType == ParamType.Body => headValue
```

Comment:
Use `Seq(headValue, _*)` instead because the list is not guaranteed to be a List